toil/src/cgcloud/toil/toil_cluster.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from cgcloud.toil.toil_box import ToilBox, ToilLeader
+from cgcloud.mesos.mesos_cluster import CreateMesosCluster
+from subprocess import check_call
+
+log = logging.getLogger( __name__ )
+shared_dir="/home/ubuntu/shared/"
+class CreateToilCluster(CreateMesosCluster):
+    def __init__( self, application ):
+        super( CreateToilCluster, self ).__init__( application )
+        self.option( '--dir_to_copy', default=None,
+                     help='The path of a local directory to distribute onto each node.' )
+
+    def run_on_creation( self, master, options ):
+        """
+        :type master: ToilMaster
+        """
+        dir = options.dir_to_copy
+        if dir:
+            check_call( ['rsync','-r','-e', 'ssh -o StrictHostKeyChecking=no', dir, "ubuntu@"+str(master.ip_address)+":"+shared_dir] )
+        log.info( "=== Launching workers ===" )
```

Comment:
Can we use the RsyncCommand or Box.rsync() here instead? 

Also, I'd rather have this prompt the user and remove the insecure StrictHostKeyChecking=no option. I plan on fixing host key checking on a global basis and leaving that option here would mean that this code wouldn't benefit from my fix in the future.